config/_default/menus.en.yaml

```diff
@@ -1,6 +1,6 @@
 header:
-  - name: Home
-    pageRef: /
+  - name: Memos
```

Comment:
I'm in favour of putting `Datasets` first, then `Memos`

Reply:
Ok